cnd/src/config/file.rs

```diff
@@ -94,7 +94,7 @@ pub struct Database {
 
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
 pub struct Bitcoin {
-    pub network: bitcoin_support::Network,
+    pub network: bitcoin::Network,
```

Comment:
Sorry, I thought I originally made a comment on that but apparently I did not...

So what is the expected configuration now for mainnet? (ie, same topic that for COMIT API and HTTP API...)